doc/decisions/3_decided/elixir_bindings.md

```diff
@@ -0,0 +1,70 @@
+# Elixir Bindings
+
+## Problem
+
+When creating Elixir bindings one needs a way to use some kind of foreign function interface.
+In Erlang (which is the basis for Elixir) such a foreign function interface exists in the form of NIFs (native implemented functions).
+The question is how this NIF interface should be implemented.
+
+## Constraints
+
+1. Elektra's C API is given.
+2. The implementation should be completed in a given time frame.
```

Comment:
Very general statement, maybe some interval (e.g. "between 20 and 50 person hours") would give a better indication of how much work is expected.

Reply:
Thanks. I made a change along those lines.